### DISCUSSION TITLE
How can I replicate WPF's DragMove in WinUI 3?

### DISCUSSION THREAD

**Discussion by GetGet99** [microsoft/WindowsAppSDK#4039] [Q&A]
So, I tried many different things for this but it does not quite work. I did try to manually move the window myself via Translation Manipulation. However, the problem is that if I drag the window to the side it does not snap and does not have snap animation. I did try to call the exact P/Invoke that WPF source code call, but the window only moves after I release my mouse. Are there any way that I can do this?

So, I would like the way to initiate dragging that is "natively handled by windows" meaning that it follows the moving behavior that Windows do. This includes the snap if I drag to the side and in Windows 11 there will be an animation as well when I drag windows on the side.

I had searched for similar discussion and I see https://github.com/microsoft/WindowsAppSDK/discussions/2962. However, it does not address the issue I have that it does not do the snapping.

**Comment by AlexanderBlackman** [comment]
I'm looking for the solution to this too.
